lib/authentication/view/sign_up_view.dart

```diff
@@ -45,25 +51,43 @@ class _SignUpViewState extends State<SignUpView> {
     _fetchFilter();
   }
 
+  void parse(TextEditingController email, TextEditingController firstName,
```

Comment:
Add a dartdoc comment (///) explaining what this does - what the regular e-mail format looks like, and how this parses it.